build.gradle

```diff
@@ -55,7 +56,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:$gradleVersion"
+        classpath "com.android.tools.build:gradle:${gradleVersion}"
```

Comment:
Is this needed?

Reply:
Yes, if you don't do this, you will get an error code